src/main/resources/db/migration/V4__add_db_table_versions.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE db_columns (
+  uuid         UUID PRIMARY KEY,
+  type         VARCHAR(64) NOT NULL,
+  created_at   TIMESTAMP NOT NULL,
+  updated_at   TIMESTAMP NOT NULL,
+  dataset_uuid UUID REFERENCES datasets(uuid),
+  name         UUID NOT NULL,
+  desscription UUID,
```

Comment:
typo ?

Reply:
alos should the type of name/description be UUID ?  shouldn't they be varchar ?

Reply:
Great catch!